schedule/yast/maintenance/qam-yast_self_update.yaml

```diff
@@ -6,5 +6,9 @@ vars:
 schedule:
   self_update:
     - installation/validate_self_update
+  additional_products: []
   add_on_product:
     - installation/add_on_product/add_maintenance_repos
```

Comment:
As discussed previously, only minimal will do the iterative, not need to use another test suite for iterative adding of maintenance repos. We use minimal because dud mechanism is only enable for x86.